README.md

```diff
@@ -197,7 +199,7 @@ __AWESOME__ apps counter: __187__ 🎉
 
 - [ ] Google Play
 - [x] [F-Droid](https://f-droid.org/packages/com.best.deskclock/)
-- [x] [GitHub](https://github.com/qw123wh/new-clock-fdroid)
+- [x] [GitHub](https://github.com/qw123wh/new-clock)
```

Comment:
I understand the change from #54, however that would mean also deleting F-Droid link, because the version is from `Added on 2022-02-03`.